### PR TITLE
feat:auth screens added

### PR DESCRIPTION
**Pull request by rajkishancode**

### CODE REVIEW

File: screens/authentication/auth.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+.input-checkbox a {
+    text-decoration: underline;
+}
+
+.label-text {
+    font-size: 1.6rem;
+    line-height: 2.4rem;
+    margin-top: 0.8rem;
+    margin-bottom: 1.6rem;
```
Comment (Naveenchandar) [inline screens/authentication/auth.css]: Please use margin shorthand.

File: screens/authentication/auth.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.input-auth {
+    max-width: 100%;
+    background-color: inherit;
+    border: 2px solid var(--grey);
+    border-radius: 5px;
+    display: block;
+    padding: 0.75rem 1rem;
+    padding: 1.2rem 1.6rem;
```
Comment (Naveenchandar) [inline screens/authentication/auth.css]: You can remove either one based on your needs. Both will not work simultaneously

**Review by prakashsakari — APPROVED** [review]
You can look into comments added by naveen, the rest looks good to me

File: screens/authentication/auth.css (hunk 2 above)
Comment (ckahub) [reply screens/authentication/auth.css]: use rem instead of px

**Review by ckahub — COMMENTED** [review]
lgtm

File: screens/authentication/auth.css (hunk 1 above)
Comment (rajkishancode) [reply screens/authentication/auth.css]: ok will change this.

File: screens/authentication/auth.css (hunk 2 above)
Comment (rajkishancode) [reply screens/authentication/auth.css]: > You can remove either one based on your needs. Both will not work simultaneously

removed previous padding.

**Pull request merged**